frontend/packages/integration-tests-cypress/views/snapshot.ts

```diff
@@ -0,0 +1,11 @@
+export namespace SnapshotDetails {
+  export const name = 'dd[data-test-selector="details-item-value__Name"]';
+  export const namespace = 'dd[data-test-selector="details-item-value__Namespace"] a';
+  export const pvc = 'dd[data-test-selector="details-item-value__PVC"] a';
```

Comment:
If changed to `data-test=...`, then this could be:
`export const pvc = cy.byTestID('details-item-value__PVC');`

Reply:
We can use `cy.byTestID` but for `PVC` we need to select the anchor tag as it contains the text we are interested in.